hudi-examples/hudi-examples-spark/pom.xml

```diff
@@ -230,6 +230,27 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- Logging dependencies -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
```

Comment:
Why do we need these here? 
These should be coming from Spark

Reply:
This log4j-1.2-api jar specifically contains implementations of the log4j api's that make it compatible with log4j2. I don't expect Spark to provide these for us. 

The other dependencies below are either logging APIs which we should declare if we directly rely on them and other bridges that were required to make the examples log properly.

Reply:
If it's for ourselves -- let's add it to "hudi-common" to avoid pasting it across the board.
If we're adding it for our dependencies, then we should not.

Reply:
This comment still holds -- let's clean up deps that are already in hudi-common and leave only `log4j-core` one